tests/e2e/cucumber/steps/ui/resources.ts

```diff
@@ -948,6 +948,7 @@ Then(
   ): Promise<void> {
     const { page } = this.actorsEnvironment.getActor({ key: stepUser })
     const resourceObject = new objects.applicationFiles.Resource({ page })
+    await page.pause()
```

Comment:
Is the pause still necessary?

Reply:
no. I'll revert it

Reply:
https://github.com/owncloud/web/pull/11809